docs/content/platform/blocks/llm.md

```
@@ -71,103 +508,293 @@ Generating creative writing, such as short stories or poetry, based on a given t
-| Input | Description |
-|-------|-------------|
-| Focus | The main topic or theme for the list to be generated |
-| Source Data | Optional data to use as a basis for list generation |
-| Model | The specific LLM to use for list generation |
-| API Key | The secret key for accessing the LLM service |
-| Max Retries | The maximum number of attempts to generate a valid list |
+| Input | Description | Type | Required |
+|-------|-------------|------|----------|
+| script_input | The text input for the script | str | Yes |
+| provider | The voice provider to use | "microsoft" | "elevenlabs" | "amazon" | No |
+| voice_id | The voice ID to use, get list of voices [here](https://docs.agpt.co/server/d_id) | str | No |
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)